miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -20,22 +20,20 @@
  *
  */
 
-
-
 /*
  * A BoxPosition represents a single position (cell) in the Box visualization (table).
  * It is responsible for handling the functionality of a cell. This includes changing the title and image
  * associated with the position.
  */
 var BoxPosition = function(opts) {
   var self = {};
-  
+
   isMultiSelectClick = function(event) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "isMultiSelectClick" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)